### PR TITLE
Modify language code functionality

### PR DESCRIPTION
**Pull request by RayceeM**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -3,7 +3,7 @@
       [path]
       (str "resources/public/js/compiled/" path))
 
-(defproject onaio/chimera "0.0.9"
+(defproject onaio/chimera "0.0.11-SNAPSHOT"
```
Comment (ivermac) [inline project.clj]: shouldn't this be `0.0.10-SNAPSHOT`, no?

**Review by ivermac — APPROVED** [review]
lgtm!

File: project.clj (hunk 1 above)
Comment (RayceeM) [reply project.clj]: there is a `0.0.10` that is currently on clojars

Comment (ivermac) [reply project.clj]: cool

**Pull request merged**